Automattic/delphin
# Domain Setup: Remove Hosts and Host Info pages

This pull request removes the former `Hosts` and `Host Info` pages which are no longer needed, thus removing stale code and reducing the size of the bundle.

### Hosts

![hosts](https://cloud.githubusercontent.com/assets/594356/21046265/21cd49a4-be04-11e6-836c-6283df1f0673.png)

### Host Info

![host info](https://cloud.githubusercontent.com/assets/594356/21046268/25ed2536-be04-11e6-81a3-d97f43985fde.png)

### Before

```
                           Asset     Size  Chunks             Chunk Names
0.bundle.fe2e3283f96baf7e6e78.js   422 kB       0  [emitted]
1.bundle.fe2e3283f96baf7e6e78.js   286 kB       1  [emitted]
2.bundle.fe2e3283f96baf7e6e78.js   125 kB       2  [emitted]
  bundle.fe2e3283f96baf7e6e78.js  2.24 MB       3  [emitted]  app
  vendor.fe2e3283f96baf7e6e78.js  2.22 MB       4  [emitted]  vendor
```

### After

```
                           Asset     Size  Chunks             Chunk Names
0.bundle.7bfa927c157f65a4db00.js   366 kB       0  [emitted]
1.bundle.7bfa927c157f65a4db00.js   286 kB       1  [emitted]
2.bundle.7bfa927c157f65a4db00.js   125 kB       2  [emitted]
  bundle.7bfa927c157f65a4db00.js  2.24 MB       3  [emitted]  app
  vendor.7bfa927c157f65a4db00.js  2.22 MB       4  [emitted]  vendor
```

#### Testing instructions

1. Run `git checkout remove/former-host-pages` and start your server, or open a [live branch](https://delphin.live/?branch=remove/former-host-pages)
2. Open the [`Login` page](http://delphin.localhost:1337/log-in)
3. Navigate to the `My Domains` page
4. Open the former [`Hosts` page](http://delphin.localhost:1337/fr/set-up-domain/YOURDOMAIN/hosts) for one of your domain
5. Check that you are presented with a 404 error page

#### Reviews

 
@Automattic/sdev-feed
